docs/uml/main/ExtEvSimulationClasses.puml

```diff
@@ -1,177 +1,149 @@
-    placeholder4 -[hidden]> EvMovement
-    ExtSimulation -[hidden]-> placeholder2
-    placeholder2 -[hidden]-> placeholder3
-    placeholder3 -[hidden]-> ExtSimAdapterData
-    placeholder2 -[hidden]> ExtData
+    node sub2{
+        interface EvDataMessageFromExt
 
+        interface DataMessageFromExt
 
-    ' RIGHT PART
-    abstract class EvMovement
-    class Arrival
-    class Departure
-    Arrival --|> EvMovement
-    Departure --|> EvMovement
-    Arrival -[hidden]> Departure
+        class ProvideArrivingEvs
 
-    interface EvModel {
-        + UUID getUuid()
-        + String getId()
-        + ComparableQuantity<Power> getSRated()
-        + ComparableQuantity<Energy> getEStorage()
-        + ComparableQuantity<Energy> getStoredEnergy()
-        + EvModel copyWith(ComparableQuantity<Energy> newStoredEnergy)
+        class RequestCurrentPrices
+
+        class RequestDepartingEvs
+
+        class RequestEvcsFreeLots
     }
 
-    ExtDataSimulation -[hidden]> Arrival
-    EvMovement *-> EvModel
-    EvModelImpl --|> EvModel
+    RequestDepartingEvs --u|> EvDataMessageFromExt
+    RequestEvcsFreeLots --u|> EvDataMessageFromExt
+    ProvideArrivingEvs --u|> EvDataMessageFromExt
+    RequestCurrentPrices --u|> EvDataMessageFromExt
 
+    DataMessageFromExt --d|> EvDataMessageFromExt
+    ExtEvData --r[norank]> EvDataMessageFromExt
+    ExtEvData --u|> ExtData
+    ExtEvSimulation -u> ExtEvData
 
-    interface EvDataMessageFromExt
-    class EvMovementsMessage {
-        - Map<UUID, List<EvMovement>> movements
-    }
-    class RequestEvcsFreeLots
-    RequestEvcsFreeLots --|> EvDataMessageFromExt
-    EvMovementsMessage --|> EvDataMessageFromExt
-    RequestEvcsFreeLots -[hidden]> EvMovementsMessage
-
-    interface EvDataResponseMessageToExt
-    class AllDepartedEvsRepsonse {
-        - Map<UUID, List<EvMovement>> movements
-    }
-    class ProvideEvcsFreeLots
-    ProvideEvcsFreeLots --|> EvDataResponseMessageToExt
-    AllDepartedEvsRepsonse --|> EvDataResponseMessageToExt
-    ExtEvData -> EvDataMessageFromExt
-    ExtEvData -> EvDataResponseMessageToExt
-    EvMovement -[hidden]-> RequestEvcsFreeLots
-    EvDataMessageFromExt -[hidden]> EvDataResponseMessageToExt
-    EvMovementsMessage -[hidden]> ProvideEvcsFreeLots
-    ProvideEvcsFreeLots -[hidden]> AllDepartedEvsRepsonse
-
-    class ScheduleDataServiceMessage {
-        - ExtEvDataService dataService
-    }
-    EvDataResponseMessageToExt -[hidden]> ScheduleDataServiceMessage
-    ExtEvData -> ScheduleDataServiceMessage
+    node sub1{
+        interface DataResponseMessageToExt
 
+        interface EvDataResponseMessageToExt
 
-    ' LEFT PART
-    interface ExtLinkInterface {
-        + ExtSimulation getExtSimulation()
-        + List<ExtDataSimulation> getExtDataSimulations()
-    }
-    ExtLinkInterface -[hidden]> ExtSimulation
-    ExtLink --|> ExtLinkInterface
+        class ProvideCurrentPrices{
+            + ProvideCurrentPrices()
```

Comment:
I would not specify the constructor, but maybe rather the field (`Map<UUID, Double> prices` here). 
Same for the classes below.